templates/max/utils/Makefile.toml

```diff
@@ -8,28 +8,30 @@ additional_profiles = [ "release" ]
 MAX_PACKAGE_NAME = { value = "median", condition = { env_not_set = ["MAX_PACKAGE_NAME"] } }
 MAX_EXT_NAME = { value = "${CARGO_MAKE_CRATE_FS_NAME}", condition = { env_not_set = ["MAX_EXT_NAME"] } }
 MAX_EXT_BASE_NAME = "${MAX_EXT_NAME}"
-PLATFORM_INSTALL_DIR = { source = "${CARGO_MAKE_RUST_TARGET_OS}", default_value = "ERROR", mapping = {"macos" = "${HOME}/Documents/Max 8/Packages", windows = "${USERPROFILE}/Documents/Max 8/Packages" } }
-INIT_INSTALL_DIR = "${PLATFORM_INSTALL_DIR}/${MAX_PACKAGE_NAME}/init/"
-EXTERNAL_INSTALL_DIR = "${PLATFORM_INSTALL_DIR}/${MAX_PACKAGE_NAME}/externals/"
+PLATFORM_INSTALL_DIR = { source = "${CARGO_MAKE_RUST_TARGET_OS}", default_value = "ERROR", mapping = { macos = "${HOME}/Documents/Max 8/Packages", windows = "${USERPROFILE}/Documents/Max 8/Packages" } }
+INSTALL_DIR = "${PLATFORM_INSTALL_DIR}/${MAX_PACKAGE_NAME}"
+EXTERNAL_INSTALL_DIR = "${INSTALL_DIR}/externals/"
```

Comment:
The makefile here is simply a copy from the `median/utils` dir, which I consider to sort of be the authoritative source. I had hoped to simply make these files a symlink but symlinks don't work with cargo-generate unfortunately.
In the mean time, I've actually created a PR: https://github.com/Cycling74/median/pull/7 that removes the utils directory from the template and simply downloads the files on demand.

Your changes are looking good but I still need to test them out (I see a comment below that someone is having some issues).. but, maybe we can apply them to the `median/utils` makefile instead?

Reply:
Sure, that would work as well!